main/ertp/ERTP-Guide/Overview.md

```diff
@@ -0,0 +1,276 @@
+# ERTP Guide
+
+[ERTP](https://agoric.com/documentation/glossary/#ertp) (*Electronic
+Rights Transfer Protocol*)
+is Agoric's token standard for transferring tokens and other digital assets in
+JavaScript. Using the
+[ERTP API](https://agoric.com/documentation/ertp/api/),
+you can easily create and use digital assets, all of which are
+transferred exactly the same way and with exactly the same security properties. 
+
+ERTP uses
+*[object capabilities](https://agoric.com/documentation/glossary/#object-capabilities)*
+to enforce access control. If your program has a reference to an
+object, it can call methods on that object. If it doesn't have a
+reference, it can't. For more on object capabilities, see
+[Chip Morningstar's post](http://habitatchronicles.com/2017/05/what-are-capabilities/).
+
+## Objects Overview
+
+![ERTP object relationships](relationships.svg)  
+
+There are eight fundamental ERTP objects, two of which are parts of
+another, and one which is used by that same object. For each entry,
+the name is linked to its section in this document. The following "&"
+symbol is linked to the term's [Glossary]() entry.
+
+- **[Amount](#amount)** **[&](https://agoric.com/documentation/glossary/#amount)**:
+  the representation of an asset, made up of a
+  *[value](#value)* *[&](https://agoric.com/documentation/glossary/#value)*  (the
+  size of the amount) and a
+  *[brand](#brand)* *[&](https://agoric.com/documentation/glossary/#brand)* (the
+  kind of asset). You can think of these as the "how many" and the
+  "what" of an asset.
+- **[AmountMath](#amountmath)** **[&]((https://agoric.com/documentation/glossary/#amountmath)**:
+Methods to do math operations on `amount` objects.
+- **[Purse](#purse)** **[&](https://agoric.com/documentation/glossary/#purse)**: an
+  object for long term `amount` object storage. 
+- **[Payment](#payment)** **[&](https://agoric.com/documentation/glossary/#payment)**:
+  an object for transfering an `amount` object to another party.
```

Comment:
No, this is very wrong. A payment is a transfer of economic value. The amount of digital assets transferred is the `amount`. But it is the assets that are transferred, not the amount. The amount is merely the description.